Sources/JavaScriptKit/ConvertibleToJSValue.swift

```diff
@@ -1,12 +1,12 @@
 import _CJavaScriptKit
 
 /// Objects that can be converted to a JavaScript value, preferably in a lossless manner.
-public protocol JSValueConvertible {
+public protocol ConvertibleToJSValue {
```

Comment:
Would be great to stay backward compatible (or "backward-compatible"? Hyphens trip me up 🤔).

```suggestion
@available(*, deprecated, renamed: "ConvertibleToJSValue")
public typealias JSValueConvertible = ConvertibleToJSValue

public protocol ConvertibleToJSValue {
```